yroots/Division.py

```diff
@@ -231,7 +234,7 @@ def division(polys, divisor_var=0, tol=1.e-10, verbose=False, polish=False, retu
 #             print(root, conditions[i])
                 
         if polish:
-            root = newton_polish(polys,root,tol = tol)
+            root = newton_polish(polys,root)
```

Comment:
This tol no longer exists. We could pass in a polish_tol potentially but it should be seperate